chains/orchestrator-relays/runtime/starlight/src/lib.rs

```diff
@@ -682,7 +681,7 @@ impl pallet_treasury::Config for Runtime {
     type RejectOrigin = EitherOfDiverse<EnsureRoot<AccountId>, Treasurer>;
     type RuntimeEvent = RuntimeEvent;
     type SpendPeriod = SpendPeriod;
-    type Burn = Burn;
+    type Burn = ();
```

Comment:
I would even put () in the spend period. In the test we can leave it hardcoded, but there is not reason to leave a spend period that does nothing right?

Reply:
Do you mean something like this (`6 days` replaced by `()`)?
```
pub const SpendPeriod: BlockNumber = runtime_common::prod_or_fast!((), 1 * MINUTES);
```
I don't think we need to keep it even in the tests.

Reply:
The thing is, since we don't have a spend period, how to test that it is disabled..

Reply:
@girazoki , just checked.
If we set SpendingPeriod to (), it will trigger `treasury.Spending` for each block, which is not good, right?
So the test will be working, but I think better to keep it as is. 
WDYT?

Reply:
I think its better to keep a period similar to polkadot, to avoid bugs in the pallet because nobody ever used a different period. So just leave the period like it was before.